homeassistant/components/growatt_server/sensor.py

```diff
@@ -88,6 +92,8 @@ class GrowattSensorEntityDescription(SensorEntityDescription, GrowattRequiredKey
         api_key="totalEnergy",
         unit_of_measurement=ENERGY_KILO_WATT_HOUR,
         device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        last_reset=dt.utc_from_timestamp(0),
```

Comment:
Please rebase this PR onto the latest `dev` branch and make use of the new state classes for this. See: <https://developers.home-assistant.io/blog/2021/08/16/state_class_total/>

```suggestion
        state_class= STATE_CLASS_TOTAL_INCREASING,
```